src/portal/e2e/db-testcases.spec.ts

```diff
@@ -0,0 +1,287 @@
+    await page.getByRole('textbox', { name: 'Comments' }).click();
+    await page.getByRole('textbox', { name: 'Comments' }).fill('test1234');
+    await page.getByRole('button', { name: 'OK' }).click();
+});
+
+test('Update User Password', async ({ page }) => {
+    // Create user
+    const username = await createUser(page);
+
+    // Login with user credentials
+    await page.getByRole('textbox', { name: 'Username' }).click();
+    await page.getByRole('textbox', { name: 'Username' }).fill(username);
+    await page.getByRole('textbox', { name: 'Password' }).click();
+    await page.getByRole('textbox', { name: 'Password' }).fill("Harbor12345");
+
+    await page.getByRole('button', { name: 'LOG IN' }).click();
+
+    // Update Password
+    await page.getByRole('button', { name: username }).click();
+    await page.getByRole('menuitem', { name: 'Change Password' }).click();
+    await page.getByRole('textbox', { name: 'Current Password*' }).click();
+    await page.getByRole('textbox', { name: 'Current Password*' }).fill('Harbor12345');
+    await page.getByRole('textbox', { name: 'New Password*' }).click();
+    await page.getByRole('textbox', { name: 'New Password*' }).fill('Test1234');
+    await page.getByRole('textbox', { name: 'Confirm Password*' }).click();
+    await page.getByRole('textbox', { name: 'Confirm Password*' }).fill('Test1234');
+    await page.getByRole('button', { name: 'OK' }).click();
+
+    // Logout after update
+    await page.getByRole('button', { name: username }).click();
+    await page.getByRole('menuitem', { name: 'Log Out' }).click();
+
+    // Login with Updated Password
+    await page.getByRole('textbox', { name: 'Username' }).click();
+    await page.getByRole('textbox', { name: 'Username' }).fill(username);
+    await page.getByRole('textbox', { name: 'Password' }).click();
+    await page.getByRole('textbox', { name: 'Password' }).fill('Test1234');
+
+    await page.getByRole('button', { name: 'LOG IN' }).click();
+
+    // Logout
+    await page.getByRole('button', { name: username }).click();
+    await page.getByRole('menuitem', { name: 'Log Out' }).click();
+});
+
+test('Edit Self Registration', async ({ page }) => {
+    // Login
+    await page.goto('/');
+    await page.getByRole('textbox', { name: 'Username' }).click();
+    await page.getByRole('textbox', { name: 'Username' }).fill('admin');
+    await page.getByRole('textbox', { name: 'Password' }).click();
+    await page.getByRole('textbox', { name: 'Password' }).fill('Harbor12345');
+
+    await page.getByRole('button', { name: 'LOG IN' }).click();
+
+    await expect(page.getByRole('link', { name: 'Configuration' })).toBeVisible();
+
+    //Select Configuration
+    await page.getByRole('link', { name: 'Configuration' }).click();
+
+    //Update self-registration Status
+    if (await page.locator('clr-checkbox-wrapper label').isChecked()) {
+        await page.locator('clr-checkbox-wrapper label').click();
```

Comment:
```suggestion
    // Uncheck self-registration Status
    if (await page.locator('clr-checkbox-wrapper label').isChecked()) {
        await page.locator('clr-checkbox-wrapper label').uncheck();
```

please update this. and all of the places.

remove AI comments and add meaningful comments which makes clear what the implementation below does.

Thanks